features/bootstrap/RestContext.php

```diff
@@ -1,18 +1,91 @@
 <?php
 
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\OAuth\AccessToken;
+use AppBundle\Entity\OAuth\Client;
+use AppBundle\OAuth\Model\AccessToken as AccessTokenModel;
+use AppBundle\OAuth\Model\Client as ClientModel;
+use AppBundle\OAuth\Model\Scope;
+use AppBundle\Repository\AdherentRepository;
+use AppBundle\Repository\OAuth\ClientRepository;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Gherkin\Node\TableNode;
 use Behatch\Context\RestContext as BehatchRestContext;
 use Behatch\HttpCall\HttpCallResultPool;
 use Behatch\HttpCall\Request;
+use Doctrine\ORM\EntityManagerInterface;
+use League\OAuth2\Server\CryptKey;
+use Ramsey\Uuid\Uuid;
+use Symfony\Component\DependencyInjection\ContainerInterface;
 
 class RestContext extends BehatchRestContext
 {
     private $httpCallResultPool;
+    private $entityManager;
+    private $privateCryptKey;
 
-    public function __construct(Request $request, HttpCallResultPool $httpCallResultPool)
-    {
+    /**
+     * @var SecurityContext
+     */
+    private $securityContext;
+
+    /**
+     * @var string|null
+     */
+    private $accessToken;
+
+    public function __construct(
+        Request $request,
+        HttpCallResultPool $httpCallResultPool,
+        EntityManagerInterface $entityManager,
+        ContainerInterface $container
+    ) {
         parent::__construct($request);
 
         $this->httpCallResultPool = $httpCallResultPool;
+        $this->entityManager = $entityManager;
+        $this->privateCryptKey = new CryptKey($container->getParameter('env(SSL_PRIVATE_KEY)'));
```

Comment:
can you try to inject the param directly in behat.yml? https://github.com/Behat/Symfony2Extension/blob/master/doc/index.rst#injecting-parameters